help/de/Help_ch_Account-Actions.xml

```diff
@@ -226,16 +224,17 @@
           </row>
 
           <row>
-            <entry namest="c1">
+            <entry>
               <para>Aufwand
               </para>
             </entry>
 <!-- FIXME source: insert footnote -->
-            <entry namest="c2">
+            <entry>
               <para>Ausgaben jeder Art, z.B. Lebensmittel, Kleidung, Steuern usw.
                 <footnote id="fn-quick-kat">
-                  <para>Wird in <quote>Quicken</quote> bzw. <quote>Lexware FinanzManager</quote> als
-                    <quote>Kategorie</quote> bezeichnet.
+                  <para>Wird in <quote>Quicken</quote> bzw. <quote>Lexware FinanzManager</quote> und anderen, nicht auf
```

Comment:
2x `<application>` statt `<quote>`

Reply:
ok.

Reply:
@fellen: Wann wird eigentlich `&app;`, `&appname;` und in welchem Fall `<application>&appnane;</application>` verwendet?

Reply:
@fellen: im dtd habe ich gerade gesehen, dass im Fließtext, also hier in der Hilfe, `&app;` verwendet werden soll und dann als `<application>GnuCash</application>` interpretiert wird.

D.h. die von mir zuvor genannte dritte Form eigentlich nicht richtig.

Reply:
Genau, sollte mir irgendwo ein `<application>GnuCash</application>`
oder `<application>&appname;</application>` … durchgerutscht sein, sofort in einem separaten commit in allen Sprachen ändern,

Im normalen Text `&app;` verwenden, in Metadaten wie `<bookinfo>` etc., aber auch vielen anderen Markups wie etwa als URL macht sowas keinen Sinn, dann also `&appname;` Ggf. zuvor testen, was besser aussieht.

Reply:
ok.